membership_file/migrations/0001_initial.py

```diff
@@ -0,0 +1,66 @@
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Member',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=63)),
+                ('tussenvoegsel', models.CharField(blank=True, max_length=15, null=True)),
+                ('last_name', models.CharField(max_length=63)),
+                ('tue_card_number', models.CharField(blank=True, max_length=15, null=True, unique=True, validators=[django.core.validators.RegexValidator(message='TU/e card numbers must only consist of exactly 7 numbers. E.g. 1234567', regex='^[0-9]{7}$')])),
+                ('external_card_number', models.CharField(blank=True, max_length=15, null=True, unique=True, validators=[django.core.validators.RegexValidator(message="External card numbers must only consist of exactly 7 numbers, followed by a hyphen (-), and ended by the 'external number' which consists of exactly 3 numbers. E.g. 1234567-123", regex='^[0-9]{7}\\-[0-9]{3}$')])),
+                ('date_of_birth', models.DateField(default=datetime.date(1970, 1, 1))),
+                ('email', models.EmailField(max_length=254, unique=True)),
+                ('phone_number', models.CharField(blank=True, max_length=16, null=True, unique=True, validators=[django.core.validators.RegexValidator(message="Phone number must be entered in the format: '+999999999'. Up to 14 digits allowed.", regex='^\\+[0-9]{8,15}$')])),
+                ('member_since', models.DateField(default=datetime.date.today)),
+                ('last_updated_date', models.DateTimeField(auto_now=True)),
+                ('user', models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='related_user', to=settings.AUTH_USER_MODEL)),
+                ('city', models.CharField(max_length=63)),
+                ('house_number', models.IntegerField(default=1, validators=[django.core.validators.MinValueValidator(1)])),
+                ('house_number_addition', models.CharField(blank=True, max_length=15, null=True)),
+                ('postal_code', models.CharField(max_length=15, validators=[django.core.validators.RegexValidator(message='Postal Codes must only consist of alphanumerical characters, spaces, and hyphens (-).', regex='^[0-9A-Za-z\\-" "]*$')])),
+                ('street', models.CharField(max_length=63)),
+                ('country', models.CharField(max_length=63)),
+                ('external_card_cluster', models.CharField(blank=True, max_length=63, null=True)),
+                ('state', models.CharField(blank=True, max_length=63, null=True)),
```

Comment:
It's explained in /membership_file/models.py, you're looking at the auto-generated migration file here! (P.s. it indeed is a (US) state/province)

Reply:
can we _not_ store an US state here? We don't have American members with American addresses (and we're not planning to expand to there soon)

Reply:
It may be entirely possible that a member moves abroad for a year (it has happened before). While in the Netherlands (or Belgium) the province is not important for an address, I'm not all too familiar with how this works in other countries. It doesn't matter if a member has the possibility to fill in their province (E.g. Noord-Brabant for Eindhoven), but it would be a shame if it  _would_ be necessary but there would be no option to do so.
Besides additional memory that is used, I don't think there are any major drawbacks.